pkg/sql/delegate/show_transactions.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package delegate
+
+import (
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqlbase"
+)
+
+const _ = `
+  id UUID,                 -- the unique ID of the transaction
+  node_id INT,             -- the ID of the node running the transaction
+  session_id STRING,       -- the ID of the session
+  start TIMESTAMP,         -- the start time of the transaction
+  txn_string STRING,       -- the string representation of the transcation
+  application_name STRING, -- the name of the application as per SET application_name
+  num_stmts INT,           -- the number of statements executed so far
+  num_retries INT,         -- the number of times the transaction was restarted
+  num_auto_retries INT     -- the number of times the transaction was automatically restarted
+`
+
+func (d *delegator) delegateShowTransactions(n *tree.ShowTransactions) (tree.Statement, error) {
+	const query = `SELECT node_id, id, application_name, num_stmts, num_retries, num_auto_retries FROM `
+	table := `crdb_internal.node_transactions`
```

Comment:
Nit -- we preface this with "".crdb_internal... so that nothing goes wrong if we aren't connected to a database and try and execute this